package com.valrTechassessment.exception

class NotFoundException: Exception("Record Not Found") {
}